src/github/interface.ts

```diff
@@ -223,6 +223,7 @@ export interface PullRequest extends Issue {
 	mergeCommitMeta?: { title: string, description: string };
 	squashCommitMeta?: { title: string, description: string };
 	suggestedReviewers?: ISuggestedReviewer[];
+	closingIssues?: Pick<Issue, 'id' | 'title' | 'number' | 'state'>[]
```

Comment:
`Pick<Issue, 'id' | 'title' | 'number' | 'state'>` is used in multiple places. Can you refactor it into a type so that it can be reused?

Reply:
I ended up doing a similar refactoring while reviewing.